src/pages/index.js

```diff
@@ -1,55 +1,74 @@
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import { useEffect, useState } from "react";
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+import ChatRoom from "../Components/ChatRoom";
 
-export default function Home() {
-  return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
+if (!firebase.apps.length) {
+  firebase.initializeApp({
+  apiKey: "AIzaSyCSq1Osue2vbeuMWIA8ErDAQwzZTHvbUE0",
```

Comment:
try to pull api keys from environment file. And try not to expose as its public repo

Reply:
ok will do